compiler/aarch64/env/OMRCPU.hpp

```diff
@@ -84,6 +84,15 @@ class OMR_EXTENSIBLE CPU : public OMR::CPU
     */
    bool isTargetWithinUnconditionalBranchImmediateRange(intptr_t targetAddress, intptr_t sourceAddress);
 
+   /**
+    * @brief Determines whether 32bit integer rotate is available
+    */
+   bool getSupportsHardware32bitRotate() { return true; }
+
+   /**
+    * @brief Determines whether 64bit integer rotate is available
+    */
+   bool getSupportsHardware64bitRotate() { return true; }
```

Comment:
I don't see a single use of these APIs in OMR. A subsequent comment mentions they are used in OpenJ9. Can we instead move these APIs to OpenJ9?

Reply:
Capability of generating instructions for rotation is independent of languages.  I think it is appropriate to implement the APIs in OMR.

Reply:
Thanks for your comments. The intention of this change is that we create the language agnostic API that asks if the rotation is available on the target CPU. I believe that `getSupportsHardwareSQRT` is one of such APIs.
https://github.com/eclipse/omr/blob/8e718c25dceb2e77bc4ead7ab61079d3b4090ab4/compiler/env/OMRCPU.hpp#L123

Reply:
@fjeremic Do you have any further comment on this PR?  Is it OK to merge this?

Reply:
Should be ok to merge.